src/app/patron/patrons-all/patrons-all.component.html

```diff
@@ -0,0 +1,59 @@
+<br>
+<div style="margin-left:60px!important" class="clr-row">
+    <div class="clr-col-lg-4 clr-col-12">
+        <a routerLink="addPatron"><button class="btn">Add Patron</button></a>
+        <a routerLink="search"><button class="btn">Search Patron</button></a>
+    </div>
+</div>
+<br>
+
+
+<div style="margin-left:60px!important" class="clr-row">
```

Comment:
move the inline style in a separate file. Here and in any other places you use inline styling